<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaPrincipal</class>
 <widget class="QMainWindow" name="VentanaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>409</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>651</width>
    <height>409</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Alineamiento de secuencias</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QTabWidget" name="tabInicio">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Ingresar Secuencias</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Secuencia1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_secuencia1">
                <property name="editable">
                 <bool>true</bool>
                </property>
                <item>
                 <property name="text">
                  <string>TCTTTGCTTTTCTTGTTTAAAATCGTACGTGTCACCCTACCCGG</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>TCTTTGCTTTTCTTGTTTAAAATCGTACGTGTCACCCTACCCGGTCTTTGCTTTTCTTGTTTAAAATCGTACGTGTCACCCTACCCGGTCTTTGCTTTTCTTGTTTAAAATCGTACGTGTCACCCTACCCGGTCTTTGCTTTTCTTGTTTAAAATCGTACGTGTCACCCTACCCGGTCTTTGCTTTTCTTGTTTAAAATCGTACGTGTCACCCTACCCGGTCTTTGCTTTTCTTGTTTAAAATCGTACGTGTCACCCTACCCGGTCTTTGCTTTTCTTGTTTAAAATCGTACGTGTCACCCTACCCGGTCTTTGCTTTTCTTGTTTAAAATCGTACGTGTCACCCTACCCGGTCTTTGCTTTTCTTGTTTAAAATCGTACGTGTCACCCTACCCGGTCTTTGCTTTTCTTGTTTAAAATCGTACGTGTCACCCTACCCGGTCTTTGCTTTTCTTGTTTAAAATCGTACGTGTCACCCTACCCGGTCTTTGCTTTTCTTGTTTAAAATCGTACGTGTCACCCTACCCGGTCTTTGCTTTTCTTGTTTAAAATCGTACGTGTCACCCTACCCGGTCTTTGCTTTTCTTGTTTAAAATCGTACGTGTCACCCTACCCGGTCTTTGCTTTTCTTGTTTAAAATCGTACGTGTCACCCTACCCGGTCTTTGCTTTTCTTGTTTAAAATCGTACGTGTCACCCTACCCGG</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Secuencia2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_secuencia2">
                <property name="editable">
                 <bool>true</bool>
                </property>
                <item>
                 <property name="text">
                  <string>TTGCTTTTCTTGTTTAAAATCGTACGTGTCGACCTACCCGGGG</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>TTGCTTTTCTTGTTTAAAATCGTACGTGTCGACCTACCCGGGGTTGCTTTTCTTGTTTAAAATCGTACGTGTCGACCTACCCGGGGTTGCTTTTCTTGTTTAAAATCGTACGTGTCGACCTACCCGGGGTTGCTTTTCTTGTTTAAAATCGTACGTGTCGACCTACCCGGGGTTGCTTTTCTTGTTTAAAATCGTACGTGTCGACCTACCCGGGGTTGCTTTTCTTGTTTAAAATCGTACGTGTCGACCTACCCGGGGTTGCTTTTCTTGTTTAAAATCGTACGTGTCGACCTACCCGGGGTTGCTTTTCTTGTTTAAAATCGTACGTGTCGACCTACCCGGGGTTGCTTTTCTTGTTTAAAATCGTACGTGTCGACCTACCCGGGGTTGCTTTTCTTGTTTAAAATCGTACGTGTCGACCTACCCGGGGTTGCTTTTCTTGTTTAAAATCGTACGTGTCGACCTACCCGGGGTTGCTTTTCTTGTTTAAAATCGTACGTGTCGACCTACCCGGGGTTGCTTTTCTTGTTTAAAATCGTACGTGTCGACCTACCCGGGGTTGCTTTTCTTGTTTAAAATCGTACGTGTCGACCTACCCGGGGTTGCTTTTCTTGTTTAAAATCGTACGTGTCGACCTACCCGGGG</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Cargar Secuencias</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLineEdit" name="lineEdit_examinar"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_examinar">
            <property name="text">
             <string>Examinar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="alinear_pushButton">
            <property name="text">
             <string>Â¡ALINEAR!</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="limpiar_pushButton">
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;Sans Serif&quot;;</string>
            </property>
            <property name="text">
             <string>Limpiar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="salida_textBrowser">
          <property name="contextMenuPolicy">
           <enum>Qt::DefaultContextMenu</enum>
          </property>
          <property name="lineWrapMode">
           <enum>QTextEdit::NoWrap</enum>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="acceptRichText">
           <bool>true</bool>
          </property>
          <property name="openExternalLinks">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>248</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="verMatriz_pushButton_2">
          <property name="text">
           <string>Ver Matriz</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup_2</string>
          </attribute>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>248</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <widget class="QMenu" name="menuExportarComo">
     <property name="title">
      <string>ExportarComo</string>
     </property>
     <addaction name="actionArchivo_FASTA"/>
     <addaction name="actionHtml"/>
    </widget>
    <addaction name="actionAbrir"/>
    <addaction name="separator"/>
    <addaction name="menuExportarComo"/>
    <addaction name="separator"/>
    <addaction name="actionGuardar"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuEdicion">
    <property name="title">
     <string>Edicion</string>
    </property>
    <addaction name="actionCopiar"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_de_estePrograma"/>
    <addaction name="actionAcerca_de_Qt"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuEdicion"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir resultados</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar Resultado Alineacion </string>
   </property>
  </action>
  <action name="actionCopiar">
   <property name="text">
    <string>Copiar</string>
   </property>
  </action>
  <action name="actionAcerca_de_estePrograma">
   <property name="text">
    <string>Acerca de estePrograma</string>
   </property>
  </action>
  <action name="actionAcerca_de_Qt">
   <property name="text">
    <string>Acerca de Qt</string>
   </property>
  </action>
  <action name="actionArchivo_FASTA">
   <property name="text">
    <string>Archivo .FASTA</string>
   </property>
  </action>
  <action name="actionHtml">
   <property name="text">
    <string>Html</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>alinear_pushButton</tabstop>
  <tabstop>limpiar_pushButton</tabstop>
  <tabstop>salida_textBrowser</tabstop>
  <tabstop>verMatriz_pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
